server/typings/models/video/video-abuse.ts

```diff
@@ -16,12 +16,12 @@ export type MVideoAbuseId = Pick<VideoAbuseModel, 'id'>
 export type MVideoAbuseVideo =
   MVideoAbuse &
   Pick<VideoAbuseModel, 'toActivityPubObject'> &
-  Use<'Video', MVideo>
+  Use<'Video', MVideoAccountLightBlacklistAllFiles>
 
 export type MVideoAbuseAccountVideo =
   MVideoAbuse &
   Pick<VideoAbuseModel, 'toActivityPubObject'> &
-  Use<'Video', MVideo> &
+  Use<'Video', MVideoAccountLightBlacklistAllFiles> &
```

Comment:
Why did you change this definition?

Reply:
`toFormattedJSON` requires `MVideoAbuseFormattable` which is called for the listing, and required by `VideoAbuseAuditView`.